### PR TITLE
Accommodated new folder changes in cisco umbrella by keeping existing one

### PR DESCRIPTION
**Pull request by v-muuppugund**
Required items, please complete
   
   Change(s):
   - A new folder has been introduced in code

   Reason for Change(s):
   - A change has been introduced by cisco umbrella

   Version Updated:
   - No

   Testing Completed:
   - Azure function
   -
image

https://ms.portal.azure.com/#view/WebsitesExtension/FunctionMenuBlade/~/monitor/resourceId/%2Fsubscriptions%2F4383ac89-7cd1-48c1-8061-b0b3c5ccfd97%2FresourceGroups%2Fv-muuppugund%2Fproviders%2FMicrosoft.Web%2Fsites%2Fumbrellab33leqczgcbku%2Ffunctions%2FciscoUmbrellaDataConn

AWS:
https://s3.console.aws.amazon.com/s3/buckets/ciscoumbrellacatdata?region=us-west-1&tab=objects

Log Analytics work space:
image
https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/419581d6-4853-49bd-83b6-d94bb8a77887/resourceGroups/v-muuppugund/providers/Microsoft.OperationalInsights/workspaces/sentinelworkspace/logs

image

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/CiscoUmbrella/Data Connectors/ciscoUmbrellaDataConn/__init__.py [hunk 1]
```diff
@@ -445,6 +448,33 @@ def parse_csv_cdfw(self, csv_file):
                 event['EventType'] = 'cloudfirewalllogs'
                 yield event
 
+    def parse_csv_fw(self, csv_file):
+        csv_reader = csv.reader(csv_file.split('\n'), delimiter=',')
+        for row in csv_reader:
+            if len(row) > 1:
+                if len(row) >= 14:
```
Comment (anki-narravula) [inline Solutions/CiscoUmbrella/Data Connectors/ciscoUmbrellaDataConn/__init__.py:455]: I believe, you would require minimum 30 columns to work below parsing. Why 14 ? I hope you are confident about the position number, maximum index you use is 29 (position 30)

**Review by anki-narravula — CHANGES_REQUESTED** [review]

File: Solutions/CiscoUmbrella/Data Connectors/ciscoUmbrellaDataConn/__init__.py (hunk 1 above)
Comment (v-muuppugund) [reply Solutions/CiscoUmbrella/Data Connectors/ciscoUmbrellaDataConn/__init__.py:455]: Yes Anki,Checked it ,Will correct it its parsing only 14 rows earlier then will cross check entire code,noticed schema is incorrect in cloud firewall logs [Log Formats and Versioning (umbrella.com)](https://docs.umbrella.com/umbrella-user-guide/docs/log-format-and-versioning#firewall)
Verdict is not present and its Action now ,will change the existing 14 rows to 30 and Verdict to Action

Comment (v-muuppugund) [reply Solutions/CiscoUmbrella/Data Connectors/ciscoUmbrellaDataConn/__init__.py:455]: As discussed, reverted the changes, as we are not sure the position of this column as it's not clear in documentation, will do once get confirmation from cisco support team.

**Review by anki-narravula — DISMISSED** [review]
Changes looks good. Approved

**Review by v-prasadboke — APPROVED** [review]

**Pull request merged**